runtime/mux.go

```diff
@@ -164,6 +165,23 @@ func (s *ServeMux) Handle(meth string, pat Pattern, h HandlerFunc) {
 	s.handlers[meth] = append([]handler{{pat: pat, h: h}}, s.handlers[meth]...)
 }
 
+
+// HandlePath allows users to configure custom path handlers.
+// refer : ../docs/_docs/inject_router.md
```

Comment:
Lets link to the docs website: https://grpc-ecosystem.github.io/grpc-gateway/docs/inject_router.html

Reply:
(This link won't work until we have released v2)